src/getDatasetHelpers.js

```diff
@@ -106,7 +114,7 @@ const joinPartsIntoPrefix = ({source, prefixParts, isNarrative = false}) => {
     // no default
   }
 
-  return [...leadingParts, ...prefixParts.filter((x) => x.length)].join("/");
```

Comment:
Introduced in https://github.com/nextstrain/nextstrain.org/commit/2857fb1cda9a144a08f11a5c70e8b56cfabf167c. This had to be removed to avoid `joinPartsIntoPrefix(splitPrefixIntoParts("http://a.b.c"))` returning `"http:/a.b.c"` (somewhat pseudocode). It may have unintended consequences for other datasets (see comment in PR re: testing0

Reply:
94f91a5 specifies that the "prefix" (to use the language of these function) doesn't specify the scheme and so we can revert this chage